call-apis-on-users-behalf/others-api/langchain-fastapi-py/README.md

```diff
@@ -49,7 +49,7 @@ Now you're ready to run the development server:
 ```bash
 source .venv/bin/activate
 uv pip install auth0_fastapi # install the auth0 fastapi package
-fastapi dev app/main.py --port 3000
+fastapi dev app/main.py
```

Comment:
Leaving to default port 8000 as it was before.

Note: you will need this while testing:
Allowed Callback URLs: http://localhost:8000/api/auth/callback
Allowed Logout URLs: http://localhost:8000
Allowed Web Origins: http://localhost:8000